.semaphore/semaphore.yml

```diff
@@ -13,18 +13,21 @@ blocks:
           matrix:
             - env_var: RUBY_VERSION
               values:
-                - 2.5.9
                 - 2.6.10
-                - 2.7.6
-                - 3.0.4
-                - 3.1.2
-                - jruby-9.3.4.0
-                - truffleruby-22.0.0.2
+                - 2.7.7
+                - 3.0.5
+                - 3.1.3
+                - 3.2.0
+                - jruby-9.3.9.0
+                - jruby-9.4.0.0
```

Comment:
If you want to use the latest release of each release series you could use `2.6, 2.7, '3.0', 3.1, 3.2, jruby-9.3, jruby-9.4, truffleruby-22, truffleruby+graalvm-22

Reply:
`truffleruby-22, truffleruby+graalvm-22` are redundant (they will both test in `--native` mode, the default), so I'd keep just `truffleruby-22`

Reply:
Or even just `truffleruby` (i.e., the latest release) so then it always tests the latest release of TruffleRuby, which is always supported

Reply:
AH, although I have no idea what syntax SemaphoreCI supports :sweat_smile: (I thought it was GitHub Actions)

Reply:
Its rbenv under the covers - so I'll play around with it.

Reply:
@eregon Its rbenv && ruby-build under the covers (which I didn't know you helped maintain - thanks btw!) - and doing just `major.minor` with that doesn't appear to be supported.  github actions must be doing some `.rbenv/versions/` symlinking under the covers.

Is doing `rbenv install major.minor` a feature request I should drop into rbenv/ruby-build ?

```
semaphore@semaphore-vm:~$ rbenv install --version
ruby-build 20221225-9-g697bcff

semaphore@semaphore-vm:~$ rbenv install 3.1
ruby-build: definition not found: 3.1

The following versions contain `3.1' in the name:
  2.3.1
  3.1.0-dev
  3.1.0-preview1
  3.1.0
  3.1.1
  3.1.2
...
```

Reply:
I would tend to think such resolution should happen before it hits ruby-build/rbenv. For instance https://github.com/ruby/setup-ruby does that on its own. Also it sounds pretty slow for a CI if it has to compile CRuby from source :sweat_smile: (but probably it does some caching?)

But maybe it makes sense for ruby-build too, feel free to file a suggestion there.

Reply:
Semaphore does that too with their `sem-version` tool - which is roughly equivalent to the github action, its what is used in the pipeline to pick the ruby version, so it uses a prebuilt one if it has one, and then delegates to ruby-build if it doesn't have it pre-built.

Reply:
I've opened a ticket with semaphore it looks like they just haven't `sem-version` to handle 3.2 patch versions yet. I'll update all of this when/if that is resolved.